sdk/core/core/test/test_http_response_parser.h

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_http_response_parser.h>
+#include <az_span.h>
+
+#include "./az_test.h"
```

Comment:
did you mean to do "./" here? it's not necessary.

Reply:
Until we solve test story, I'm ok with this. It's easier than altering CMake